### ISSUE TITLE
Questions regarding: Fig Caption and Position Absolute (width)

### ISSUE THREAD

**Issue by dasi0075** [dasi0075/hot-grid-dle#1]
Hi @thomasjbradley
Sorry to bother I am trying to get ahead with my coding this week and have two questions:
1. First I had done figure and fig caption but then my photos came up with a margin around just like the previous assignment so I changed for div, it looks similar but markbot expects figure and fig caption. How to take the margin off then?

2. With position absolute I got to make the look similar but I remember there was a way to stretching the caption and image together (see screenshot) when I stretch my screen my caption remains small

<img width="1160" alt="screen shot 2019-01-22 at 8 02 08 am" src="https://user-images.githubusercontent.com/43067752/51537457-a5bc2800-1e1c-11e9-8fd1-5287706eda45.png">

**Comment by thomasjbradley** [comment]
Yes, use a `<figure>` & `<figcaption>`—and yes, get rid of the margins and stuff.

I don’t believe my captions change width at all—I think they’re a fixed size.
